zyngine/zynthian_engine_pianoteq.py

```diff
@@ -134,19 +135,19 @@ def fix_pianoteq_config():
 				logging.debug("Fixing Audio Setup")
 				for devicesetup in audio_setup_node.iter('DEVICESETUP'):
 					devicesetup.set('deviceType','JACK')
-					devicesetup.set('audioOutputDeviceName','Auto-connect ON')
-					devicesetup.set('audioInputDeviceName','Auto-connect ON')
-					devicesetup.set('audioDeviceRate','44100')
+					devicesetup.set('audioOutputDeviceName','Auto-connect OFF')
```

Comment:
This is a good point. It should prevent strange effects when loading snapshots with pianoteq+effects layers.